source/includes/introduction.md

```diff
@@ -13,3 +13,42 @@ Highline's API is organized around REST. Our API has predictable, resource-orien
 ## Sandbox
 
 Our sandbox environment at `https://sandbox.highline.co` provides all the functionality of `https://api.highline.co` with test account credentials. The sandbox API credentials is also available in the product settings page at `https://app.highline.co/manage-product`.
+
```

Comment:
I'd add the production link url here as well. 

Highline-Link's sandbox url at `https://stg-link.highline.co/highline-link.js` provides all the functionality of `https://link.highline.co/v1/highline-link.js` with these credentials: